src/components/widget/DraggableWidget.tsx

```diff
@@ -68,21 +62,12 @@ export default class DraggableWidget extends React.PureComponent<
   }
 
   render() {
-    const {
-      locationX,
-      locationY,
-      moved,
-    } = this.state;
-
     return (
       <Widget
         {...this.props}
-        locationX={locationX}
-        locationY={locationY}
-        calculateInverted={!moved}
       >
         {this.props.children}
```

Comment:
side note: this is already passed in `{...this.props}`, so `<Widget {...this.props} />` should work fine